Dockerfile.dev

```diff
@@ -1,14 +1,10 @@
-FROM ubuntu:latest
-
-RUN apt update && apt upgrade
-RUN apt install python3.10 pip -y
+FROM python:3.10.9
+RUN apt-get update && pip install --upgrade pip
 
+WORKDIR /opt/root-pythia/src
 COPY requirements.txt /tmp/requirements.txt
 COPY requirements-dev.txt /tmp/requirements-dev.txt
-RUN python3.10 -m pip install --upgrade pip
-RUN python3.10 -m pip install -r /tmp/requirements-dev.txt
+RUN pip install -r /tmp/requirements-dev.txt
 
-#RUN useradd --create-home --shell /bin/bash dev
-#USER dev
-WORKDIR /opt/root-pythia
-CMD ["bash"]
+ENTRYPOINT ["python3"]
+CMD ["main.py"]
```

Comment:
You're still heavily modifying the dev mode, is it done on purpose?

Reply:
Not really, I forgot to restore this file. I still added minor updates regarding the image we use.